public/js/application.js

```diff
@@ -1,7 +1,17 @@
 $(document).ready(function() {
-  // This is called after the document has loaded in its entirety
-  // This guarantees that any elements we bind to will exist on the page
-  // when we try to bind to them
+  // delete list
+  $('.delete-list').click(function(event) {
+      event.preventDefault();
 
-  // See: http://docs.jquery.com/Tutorials:Introducing_$(document).ready()
+      var $current_list = $(this).parent().parent();
```

Comment:
I believe jQuery requests that you do not declare variables with `$` preceding them, in order to not accidentally get your references mixed up.